### PR TITLE
made the installer simpler

### PR DESCRIPTION
**Pull request by optimalcomrad**
Made the installer simpler. You don't need -get anymore, to my understanding. And changed some dpkg's to apt, because it gets missing dependencies, dpkg don't

### CODE REVIEW

**Review by ghost — CHANGES_REQUESTED** [review]
impossible gray toothbrush hungry dam wild bag rotten intelligent future

This post was mass deleted and anonymized with Redact

File: installer.sh [hunk 1]
```diff
@@ -7,29 +7,31 @@ if [ $(id -u) -ne 0 ]; then
 fi
 
 # For upgrades and sanity check, remove any existing i2p.list file
-rm -f /etc/apt/sources.list.d/i2p.list
+rm -r /etc/apt/sources.list.d/i2p.list
```
Comment (Und3rf10w) [inline installer.sh:10]: Incorrect and improper command

File: installer.sh [hunk 2]
```diff
@@ -7,29 +7,31 @@ if [ $(id -u) -ne 0 ]; then
...
 # Configure and install the .deb
-dpkg-deb -b kali-anonsurf-deb-src/ kali-anonsurf.deb # Build the deb package
-dpkg -i kali-anonsurf.deb || (apt-get -f install && dpkg -i kali-anonsurf.deb) # this will automatically install the required packages
+apt install kali-anonsurf-deb-src/ kali-anonsurf.deb # Build the deb package
+apt install kali-anonsurf.deb || (apt -f install && apt install  kali-anonsurf.deb) # this will automatically install the required packages
 
-exit 0
+echo "Done"
+
+exit0
```
Comment (Und3rf10w) [inline installer.sh:37]: Broken exit statement

**Comment by Und3rf10w** [comment]
This PR contains commands that will break the package. Rejected.

**Pull request closed without merging**